src/servicesHost.ts

```diff
@@ -137,30 +153,81 @@ export function makeServicesHost(
     trace: log.log,
     log: log.log,
 
-    /* Unclear if this is useful
```

Comment:
Eheh it was 😅

Reply:
I know, right? How could I imagine a comment like that wouldn't come back to bite me 😄